README.md

```diff
@@ -15,4 +15,13 @@ Join the [brainhack slack](https://brainhack-slack-invite.herokuapp.com/) and th
 Breakfast (8:30 am) and lunch are included. The training sessions will run from 9 am to 5 pm both days. All training sessions will be at the Groupe Maurice amphitheatre, at the [centre de recherche de l'institut de gériatrie de Montréal](https://goo.gl/maps/ouhdXKKWtko). 4545 Queen Mary Rd, Montreal, QC H3W 1W6, Canada. Metro station: snowdon (orange/blue lines), côte-des-neiges (blue line). 
 ![CRIUGM](criugm.jpg)
 
-
+### Usage
+
+To use the docker image, first after cloning the repository and cd to it, build it :
+```
+sudo docker build --tag=introml .
+```
+You can now run the container :
+```
+sudo docker run -p 8888:8888 -it introml jupyter notebook --allow-root --no-browser --ip=0.0.0.0
```

Comment:
why `--allow-root` ? I thought the jovyan user was there to avoid running as root.

Reply:
I already had issues when not using `--allow-root`, specially because jupyter seems to write files in `/usr`. I am not really aware of what it is doing in deep but without it it is not working :/
For `docker`you don't have choice to run as root, it is mostly for `singularity`

Reply:
And many docker users are using this option, see official tensorflow docker :
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/dockerfiles/dockerfiles

Reply:
I removed the option allow-root, but I confirmed that it would not run in binder but was just an instruction on how to use it locally,